src/select/select.js

```diff
@@ -110,6 +110,9 @@ class Select extends React.Component<PropsT, SelectStateT> {
     if (this.props.autoFocus) {
       this.focus();
     }
+    if (this.props.openOnMount) {
+      this.setState({isOpen: true});
```

Comment:
Can this be done in `constructor()`? This is adding a second re-render.

Reply:
oh right - i knew this felt wrong